charts/app-config/values-integration.yaml

```
@@ -143,16 +143,20 @@ govukApplications:
       hosts:
         - name: assets-origin.{{ .Values.k8sExternalDomainSuffix }}
           paths:
+            - path: /assets/
```

Might be worth commenting these so it's clear that it's not drift from prod/staging, like:
```suggestion
            - path: /assets/  # Integration only.
```

---

Thanks.  I've added this comment to all 4 lines.